docs/swarm/swarm-tutorial/create-swarm.md

```diff
@@ -25,6 +25,8 @@ node. For example, the tutorial uses a machine named `manager1`.
     docker swarm init --advertise-addr <MANAGER-IP>
     ```
 
+    >**Note:** If you are using Docker for Mac or Docker for Windows to test single-node swarm, simply run `docker swarm init` with no arguments. There is no need to specify a ` --advertise-addr` in this case. To learn more, see the setup topic on [Using Docker for Mac or Docker for Windows](index.md#using-docker-for-mac-or-docker-for-windows) with Swarm mode.
```

Comment:
wrap 80 chars
s/a --advertise/an --advertise

nit: suggested revision

If you are using Docker for Mac or Docker for Windows to test single-node swarm, you don't need  to specify an ` --advertise-addr`. Run `docker swarm init` with no arguments...